### PR TITLE
Enable interop debugging for Windows amd64.

### PR DESCRIPTION
**Pull request by mikem8361**
Defining FEATURE_INTEROP_DEBUGGING is all that was needed.

Simple testing with VS attaching with both native/.NET Core debugging.  More testing will be ported/enabled in the debuggertests repo under https://github.com/dotnet/debuggertests/issues/247.

### CODE REVIEW

File: clrdefinitions.cmake [hunk 1]
```diff
@@ -131,6 +131,9 @@ if(NOT CMAKE_SYSTEM_NAME STREQUAL NetBSD)
 endif(NOT CMAKE_SYSTEM_NAME STREQUAL NetBSD)
 add_definitions(-DFEATURE_HOST_ASSEMBLY_RESOLVER)
 add_definitions(-DFEATURE_ICASTABLE)
+if (WIN32)
+add_definitions(-DFEATURE_INTEROP_DEBUGGING)
```
Comment (gregg-miskelly) [inline clrdefinitions.cmake]: Do you need any logic to disable this on ARM/ARM64?

**Comment by danwalmsley** [comment]
how do we enable this in the csproj file or from visual studio?

**Comment by mikem8361** [comment]
@gregg-miskelly do you know?

**Comment by gregg-miskelly** [comment]
@danwalmsley: I just tried it and found:
1. There isn't any good way to hack a CLI-style project to do this. You can hack an old school .csproj, but this is rather moot due to my second discovery.
2. Launch doesn't actually work correctly. In launch, our native debugger leaves some threads suspended until we receive certain events, and these events aren't going to come for CoreCLR scenarios.

I will file a bug and fix my second finding. In the mean time you can either attach using interop, or use the old work arounds of switching back and forth between native-only and managed-only debugging.

I know the .NET Core project system folks have a bug to properly expose this in their debug settings page, but I can't seem to find it at the moment.

**Pull request merged**